nva-datamodel-java/src/test/java/no/unit/nva/model/file/FileModelTest.java

```diff
@@ -172,6 +170,48 @@ void shouldNotBeVisibleForNonOwnerWhenUnpublished() throws JsonProcessingExcepti
         assertThat(unmapped.isVisibleForNonOwner(), equalTo(false));
     }
 
+    @Test
+    void objectMapperShouldSerializeAndDeserializePublishedVersion() throws JsonProcessingException {
+        var unpublishedFile = new UnpublishedFile(UUID.randomUUID(), randomString(), randomString(), 10L, null,
+                                                  false, false, null, null, randomString());
+        var unpublishedFileString = unpublishedFile.toString();
+        var publicationAfterRoundTrip = dataModelObjectMapper.readValue(unpublishedFileString, UnpublishedFile.class);
+        assertThat(publicationAfterRoundTrip.getPublisherVersion(), is(equalTo(PublisherVersion.ACCEPTED_VERSION)));
+    }
+
+    @Test
```

Comment:
Mark the test as \@Deprecated too to make the functionality easier to find to remove